server/api/code/lacity_data_api/services/utilities.py

```diff
@@ -23,20 +23,13 @@ async def build_cache():
     #     await council.get_open_request_counts(i)
 
     # get results for past week
-    await service_request.get_filtered_requests(
-        date.today() - timedelta(days=7.0),
-        date.today(),
-        list(types),
-        list(councils),
-    )
-
-    # get results for past month
-    await service_request.get_filtered_requests(
-        date.today() - relativedelta(months=1),
-        date.today(),
-        list(types),
-        list(councils),
-    )
+    for day in range(7):
```

Comment:
Hey, so this will make 7 requests, but we want each request to be for a single day only.

Basically, we want to make sure that the cache is warm when the client requests data from the server. Therefore, when we build the cache, we need to make sure that the cache entries match what the client expects. The client makes the calls to the API [here](https://github.com/hackforla/311-data/blob/dev/client/components/Map/index.js#L220), so we want to call `get_filtered_requests` in the same manner.